src/Azure/AzureAD/Authentication.AzureAD.UI/src/Areas/AzureAD/Controllers/AccountController.cs

```diff
@@ -11,6 +12,7 @@ namespace Microsoft.AspNetCore.Authentication.AzureAD.UI.AzureAD.Controllers.Int
     [AllowAnonymous]
     [Area("AzureAD")]
     [Route("[area]/[controller]/[action]")]
+    [Obsolete("This is obsolete and will be removed in a future version. Use Microsoft.Identity.Web instead. See https://aka.ms/ms-identity-web.")]
```

Comment:
I don't think this does anything meaningful since users don't use this type directly.

Reply:
See https://github.com/dotnet/aspnetcore/pull/24065#discussion_r457556081. Even internal types produce obsolete warnings if they use obsolete public APIs. We either have to mark it obsolete or add pragmas.